test/WebJobs.Script.Tests/ManagedDependencies/hostNoManagedDependencies.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Create these files in temp directory instead of checking them in.

Reply:
Hi Pragna, if there any reason to not check-in the test host.json file?

Reply:
For unit tests, we should avoid reading from files. For E2E test it makes sense to read from the host.json. These tests do not have to read from a file. They should build json object.